pom.xml

```diff
@@ -175,7 +182,8 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>scm-api</artifactId>
-        <version>2.0.8</version>
+        <version>2.2.6</version>
+        <scope>test</scope>
```

Comment:
Updated to fix `RequireUpperBoundDeps` caused for updating to workflow-support-3.0:
```
Require upper bound dependencies error for org.jenkins-ci.plugins:scm-api:2.0.8 paths to dependency are:
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins:scm-api:2.0.8
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.32
    +-org.jenkins-ci.plugins:scm-api:2.0.7
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.32
    +-org.jenkins-ci.plugins:scm-api:2.0.7
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-support:3.0
    +-org.jenkins-ci.plugins:scm-api:2.2.6
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-api:2.28
    +-org.jenkins-ci.plugins:scm-api:2.0.8
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-api:2.28
    +-org.jenkins-ci.plugins:scm-api:2.0.8
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-multibranch:2.10
    +-org.jenkins-ci.plugins:scm-api:2.0.1
and
+-org.jenkins-ci.plugins:copyartifact:1.43-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-multibranch:2.10
    +-org.jenkins-ci.plugins:branch-api:2.0.0
      +-org.jenkins-ci.plugins:scm-api:2.0.1
```

Reply:
`<scope>test</scope>` looks improperly removed in 0ecbe363bdf768524e9d7406f03543af13e71838

Reply:
Note that adding a test scope changes the scope if there is a plugin dependency which actually requires it. It will not impact users, because Jenkins relies on the update center metadata which will be fine. But it may impact stability of tests relying on this API

Reply:
I might be able to remove this dependency itself as I don’t think copyartifact directly depending to scm-api even in tests. I’ll do that as another issue.
Fortunately as far as seeing the output of `RequireUpperBoundDeps` there looks no path to scm-api in the compile scope (all workflow-xxx plugins are only for tests).